proto/data_test.go

```diff
@@ -219,11 +219,11 @@ func TestPrevKey(t *testing.T) {
 }
 
 func TestKeyString(t *testing.T) {
-	if KeyMax.String() != "\xff..." {
+	if KeyMax.String() != "\\xff..." {
```

Comment:
I think we should move to an encoding where there is a well defined "infinity key". Bigtable had this. Google has open-sourced the encoding: https://github.com/GoogleCloudPlatform/DataflowJavaSDK/blob/master/sdk/src/main/java/com/google/cloud/dataflow/sdk/runners/worker/OrderedCode.java. You'll notice a similarity to our `{Encode,Decode}Bytes` encoding.